<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor_file_converter_basedialog</class>
 <widget class="QDialog" name="monitor_file_converter_basedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>524</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Schedule: Automatic Data Insertion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="pixmapLabel">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="pixmap">
        <pixmap resource="../../gex.qrc">:/gex/icons/folder_convert.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDataPump">
       <property name="font">
        <font>
         <family>Ms Sans Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Task: Convert files to STDF or CSV</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>153</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_OkCancel">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelGeneral">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Specify general Converter settings:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabelTitle">
           <property name="text">
            <string>Title:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="LineEditTitle">
           <property name="toolTip">
            <string>Enter task title!</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabelDataSource">
           <property name="text">
            <string>Data source:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="LineEditInputPath">
             <property name="toolTip">
              <string>Import file from this path...e.g.: \\server\prod_files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonBrowseInputFolder">
             <property name="toolTip">
              <string>Pick the folder to read...</string>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEditInputFileExtensions">
             <property name="toolTip">
              <string>List of files and/or extensions to import.E.g:  *.atd;*.atdf;*.csv;*.pcm;*.gdf;*.std;*.stdf;*.wat;*.zip;*.gz;*.tar;*.Z</string>
             </property>
             <property name="text">
              <string>*.atd;*.atdf;*.csv;*.pcm;*.gdf;*.std;*.stdf;*.wat;*.zip;*.gz;*.tar;*.Z;*.txt</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QComboBox" name="comboBoxTaskPriority">
               <property name="toolTip">
                <string>How often will this task be executed (priority)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="TextLabelFileOrder">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>70</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Specify the order of file loads into the database target...</string>
               </property>
               <property name="text">
                <string>Sort files by:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxTaskFileOrder">
               <property name="toolTip">
                <string>Specify the order of file loads into the database target...</string>
               </property>
               <item>
                <property name="text">
                 <string>Date/Time - oldest to newest</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Date/Time - newest to oldest</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>FileName - alphabetical order</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBoxFormat">
             <item>
              <property name="text">
               <string>STDF</string>
              </property>
              <property name="icon">
               <iconset resource="../../gex.qrc">
                <normaloff>:/gex/icons/gex_application.png</normaloff>:/gex/icons/gex_application.png</iconset>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spreadsheet CSV</string>
              </property>
              <property name="icon">
               <iconset resource="../../gex.qrc">
                <normaloff>:/gex/icons/csv_spreadsheet.png</normaloff>:/gex/icons/csv_spreadsheet.png</iconset>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ATDF</string>
              </property>
              <property name="icon">
               <iconset resource="../../gex.qrc">
                <normaloff>:/gex/icons/gex_application.png</normaloff>:/gex/icons/gex_application.png</iconset>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxTimeStampName">
             <property name="toolTip">
              <string>Check if you want the task to be triggered at a given time</string>
             </property>
             <property name="text">
              <string>Unique name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="TextLabelFileType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>File type:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TextLabelPriorityLevel">
           <property name="text">
            <string>Priority level:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="labelDataType">
           <property name="text">
            <string>Output format:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="TextLabelDestination">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>17</height>
            </size>
           </property>
           <property name="text">
            <string>Destination:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="LineEditOutputPath">
             <property name="toolTip">
              <string>Import file from this path...e.g.: \\server\prod_files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonBrowseOutputFolder">
             <property name="toolTip">
              <string>Pick the folder to read...</string>
             </property>
             <property name="text">
              <string>Browse...</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="TextLabelTaskFrequency">
           <property name="text">
            <string>Task frequency:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QComboBox" name="comboBoxTaskFrequency">
             <property name="toolTip">
              <string>How often will this task be executed (frequency)</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="TextLabelDayOfWeek">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Day of week:</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxTaskFrequencyDayOfWeek">
               <property name="toolTip">
                <string>Specify which Day of the week...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="poRunScript">
           <property name="text">
            <string>Run script</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QComboBox" name="poRunningMode">
             <item>
              <property name="text">
               <string>On failure</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>On completion</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="poScriptToRunLabel">
           <property name="text">
            <string>Script path :</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="poScriptToRunLinedit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="poScriptToRunBrowser">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../gex.qrc">
               <normaloff>:/gex/icons/file_open.png</normaloff>:/gex/icons/file_open.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Housekeeping">
      <attribute name="title">
       <string>Housekeeping</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelHousekeeping">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Specify what to do with the files after processing by the Converter:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="TextLabelOnSuccess">
           <property name="text">
            <string>On success:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxTaskPostConversion">
           <property name="toolTip">
            <string>What to do with successfully processed files...</string>
           </property>
           <item>
            <property name="text">
             <string>Rename processed file to '.read'</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Delete data files processed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Move files to folder...</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEditPostConversion">
           <property name="toolTip">
            <string>Target Folder</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButtonBrowsePostConversion">
           <property name="toolTip">
            <string>Pick target folder...</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TextLabelOnException">
           <property name="text">
            <string>On exception:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxTaskPostConversionFailure">
           <property name="toolTip">
            <string>What to do with rejected files...</string>
           </property>
           <item>
            <property name="text">
             <string>Rename rejected files to '.bad'</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Move rejected files to folder...</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEditPostConversionFailure">
           <property name="toolTip">
            <string>Target Folder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pushButtonBrowsePostConversionFailure">
           <property name="toolTip">
            <string>Pick target folder...</string>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>204</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>LineEditTitle</tabstop>
  <tabstop>LineEditInputPath</tabstop>
  <tabstop>pushButtonBrowseInputFolder</tabstop>
  <tabstop>comboBoxFormat</tabstop>
  <tabstop>buttonBox_OkCancel</tabstop>
  <tabstop>comboBoxTaskPostConversion</tabstop>
  <tabstop>lineEditPostConversion</tabstop>
  <tabstop>pushButtonBrowsePostConversion</tabstop>
  <tabstop>comboBoxTaskPostConversionFailure</tabstop>
  <tabstop>lineEditPostConversionFailure</tabstop>
  <tabstop>pushButtonBrowsePostConversionFailure</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../gex.qrc"/>
 </resources>
 <connections/>
</ui>
